repo.yml

```diff
@@ -0,0 +1 @@
+type: node-cli
```

Comment:
More of a comment for another PR; but since we have the file, we could start recording upstream dependencies in here, this would enable nested changelogs in the CLI. https://github.com/balena-io/pinejs/blob/master/repo.yml shows how these can be setup (requires some setup on the upstream repos as well)

Reply:
I see, thanks! It sounds like this would complement an effort by @imrehg in adding the `.versionbot/CHANGELOG.yml` file. I think it will be great to have more informative changelogs.